Grbl_Esp32/src/Stepper.cpp

```diff
@@ -414,26 +324,26 @@ static void stepper_pulse_func() {
         }
     }
 
-#ifdef USE_I2S_STEPS
-    if (current_stepper == ST_I2S_STREAM) {
-        //
-        // Generate pulse (at least one pulse)
-        // The pulse resolution is limited by I2S_OUT_USEC_PER_PULSE
-        //
-        i2s_out_push_sample(pulse_microseconds->get() / I2S_OUT_USEC_PER_PULSE);
-        set_stepper_pins_on(0);  // turn all off
-        return;
-    }
-#endif
-#ifdef USE_RMT_STEPS
-    return;
-#else
-    // wait for step pulse time to complete...some of it should have expired during code above
-    while (esp_timer_get_time() - step_pulse_start_time < pulse_microseconds->get()) {
-        NOP();  // spin here until time to turn off step
+    switch (current_stepper) {
+        case ST_I2S_STATIC:
```

Comment:
Behavior for ST_I2S_STATIC == ST_I2S_STREAM ? I cannot tell... it is different from the original code.

Reply:
This needs some careful attention.  I don't yet fully comprehend how this I2S code plays out.

Reply:
ST_I2S_STATIC mode uses a timer interrupt for the pulse generation.
So the case should be the same as ST_TIMED.
`case ST_I2S_STATIC:` should be moved before `case ST_TIMED:` and fallthrough.